src/librustdoc/html/static/themes/light.css

```diff
@@ -27,10 +27,6 @@ h2, h3:not(.impl):not(.method):not(.type):not(.tymethod), h4:not(.method):not(.t
 	border-bottom-color: #DDDDDD;
 }
 
-.in-band {
```

Comment:
Was this class being used? Why was it removed from both themes?

Reply:
The headers had the `in-band` class set, but because this class sets the background color to the same background color of the page, it is not used.